From e8d32ca29ae2b64124ab5f05cd8bc3c643669208 Mon Sep 17 00:00:00 2001
From: Jason Liu <r64343@freescale.com>
Date: Mon, 9 Jun 2014 15:58:10 +0800
Subject: [PATCH 449/505] fix and cleanup the linux-test build

Replace the IMX6S with IMX6SL and add the IMX6SX support

Signed-off-by: Jason Liu <r64343@freescale.com>
---
 test/gpu/Makefile               | 2 +-
 test/max8660_pmic_test/Makefile | 2 +-
 test/mxc_cec_test/Makefile      | 2 +-
 test/mxc_i2c_test/Makefile      | 2 +-
 test/mxc_ipudev_test/Makefile   | 2 +-
 test/mxc_mlb_test/Makefile      | 2 +-
 test/mxc_pf_test/Makefile       | 2 +-
 test/mxc_pmic_test/Makefile     | 2 +-
 test/mxc_rng_test/Makefile      | 2 +-
 test/mxc_sahara_test/Makefile   | 2 +-
 test/mxc_scc/Makefile           | 2 +-
 test/mxc_scc2/Makefile          | 2 +-
 test/mxc_v4l2_test/Makefile     | 4 ++--
 test/pxp_lib_test/Makefile      | 2 +-
 test/pxp_v4l2_test/Makefile     | 2 +-
 15 files changed, 16 insertions(+), 16 deletions(-)

diff --git a/test/gpu/Makefile b/test/gpu/Makefile
index b03c3a0..40c2eea 100644
--- a/test/gpu/Makefile
+++ b/test/gpu/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which want this test case
-INCLUDE_LIST:=IMX6Q IMX6S
+INCLUDE_LIST:=IMX6Q IMX6SL IMX6SX
 
 ifeq ($(PLATFORM),$(findstring $(PLATFORM),$(INCLUDE_LIST)))
 OBJS = $(OBJDIR)/gpu.sh
diff --git a/test/max8660_pmic_test/Makefile b/test/max8660_pmic_test/Makefile
index dac5b0e..3c2684b 100644
--- a/test/max8660_pmic_test/Makefile
+++ b/test/max8660_pmic_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX51 IMX37_3STACK IMX31_3STACK IMX31ADS IMX27_3STACK IMX27ADS IMX25_3STACK IMX6Q IMX6S"
+EXCLUDE_LIST:="IMX51 IMX37_3STACK IMX31_3STACK IMX31ADS IMX27_3STACK IMX27ADS IMX25_3STACK IMX6Q IMX6SL IMX6SX"
 
 
 LTP_DIR = $(TOPDIR)/test/mxc_pmic_test/
diff --git a/test/mxc_cec_test/Makefile b/test/mxc_cec_test/Makefile
index 282d37e..98128fa 100755
--- a/test/mxc_cec_test/Makefile
+++ b/test/mxc_cec_test/Makefile
@@ -2,7 +2,7 @@
 # Makefile for MediaLB driver test app.
 #
 # list of platforms which did not want this test case
-INCLUDE_LIST:= IMX6Q IMX6S
+INCLUDE_LIST:= IMX6Q
 
 INC += -I$(LINUXPATH)/include
 CFLAGS += -lpthread -lcec -lrt
diff --git a/test/mxc_i2c_test/Makefile b/test/mxc_i2c_test/Makefile
index 4424551..a25604f 100644
--- a/test/mxc_i2c_test/Makefile
+++ b/test/mxc_i2c_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX35_3STACK IMX25_3STACK IMX6Q IMX6S"
+EXCLUDE_LIST:="IMX35_3STACK IMX25_3STACK IMX6Q IMX6SL IMX6SX"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = \
diff --git a/test/mxc_ipudev_test/Makefile b/test/mxc_ipudev_test/Makefile
index 9062c7c..7cdc6ee 100644
--- a/test/mxc_ipudev_test/Makefile
+++ b/test/mxc_ipudev_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which want this test case
-INCLUDE_LIST:=IMX51 IMX53 IMX6Q IMX6S
+INCLUDE_LIST:=IMX51 IMX53 IMX6Q
 
 SRCS = mxc_ipudev_test.c utils.c
 LIBS += -lm -lrt
diff --git a/test/mxc_mlb_test/Makefile b/test/mxc_mlb_test/Makefile
index 10cb17a..2058bcb 100644
--- a/test/mxc_mlb_test/Makefile
+++ b/test/mxc_mlb_test/Makefile
@@ -2,7 +2,7 @@
 # Makefile for MediaLB driver test app.
 #
 # list of platforms which did not want this test case
-EXCLUDE_LIST:= IMX21ADS IMX27ADS IMX31ADS IMX32ADS IMX37_3STACK IMX25_3STACK IMX6S
+EXCLUDE_LIST:= IMX21ADS IMX27ADS IMX31ADS IMX32ADS IMX37_3STACK IMX25_3STACK IMX6SL
 
 INC += -I$(LINUXPATH)/include
 
diff --git a/test/mxc_pf_test/Makefile b/test/mxc_pf_test/Makefile
index ff9bb15..6c80940 100644
--- a/test/mxc_pf_test/Makefile
+++ b/test/mxc_pf_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX27ADS IMX37_3STACK IMX51 IMX25_3STACK IMX6Q IMX6S"
+EXCLUDE_LIST:="IMX27ADS IMX37_3STACK IMX51 IMX25_3STACK IMX6Q IMX6SL IMX6SX"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/autorun-pf.sh $(OBJDIR)/mxc_pf_test.out
diff --git a/test/mxc_pmic_test/Makefile b/test/mxc_pmic_test/Makefile
index 891b3ce..b70c0ad 100644
--- a/test/mxc_pmic_test/Makefile
+++ b/test/mxc_pmic_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX37_3STACK IMX35_3STACK IMX51 IMX6Q IMX6S"
+EXCLUDE_LIST:="IMX37_3STACK IMX35_3STACK IMX51 IMX6Q IMX6SL IMX6SX"
 
 LTP_DIR = ../
 LOADLIBES+= -L$(LTP_DIR)lib -lltp -lpthread -lm
diff --git a/test/mxc_rng_test/Makefile b/test/mxc_rng_test/Makefile
index 2761ff7..c3ea840 100644
--- a/test/mxc_rng_test/Makefile
+++ b/test/mxc_rng_test/Makefile
@@ -5,7 +5,7 @@
 #****************************************************************************/
 
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX27ADS IMX51 IMX6Q IMX6S"
+EXCLUDE_LIST:="IMX27ADS IMX51 IMX6Q IMX6SL IMX6SX"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = apitest rng_test move
diff --git a/test/mxc_sahara_test/Makefile b/test/mxc_sahara_test/Makefile
index c0d8482..a49c834 100644
--- a/test/mxc_sahara_test/Makefile
+++ b/test/mxc_sahara_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:= IMX21ADS IMX31ADS IMX31_3STACK IMX35_3STACK IMX37_3STACK IMX25_3STACK IMX50 IMX6Q IMX6S
+EXCLUDE_LIST:= IMX21ADS IMX31ADS IMX31_3STACK IMX35_3STACK IMX37_3STACK IMX25_3STACK IMX50 IMX6Q IMX6SL IMX6SX
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
  OBJS= apitest
diff --git a/test/mxc_scc/Makefile b/test/mxc_scc/Makefile
index 660a007..c1808fe 100644
--- a/test/mxc_scc/Makefile
+++ b/test/mxc_scc/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX21ADS MXC30031ADS IMX233 IMX37_3STACK IMX51 IMX50 IMX6Q IMX6S"
+EXCLUDE_LIST:="IMX21ADS MXC30031ADS IMX233 IMX37_3STACK IMX51 IMX50 IMX6Q IMX6SL IMX6SX"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/scc_test
diff --git a/test/mxc_scc2/Makefile b/test/mxc_scc2/Makefile
index 2a1dbd4..9cb6b2e 100644
--- a/test/mxc_scc2/Makefile
+++ b/test/mxc_scc2/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX21ADS IMX27ADS IMX31ADS IMX31_3STACK IMX35_3STACK IMX25_3STACK IMX233 IMX50 IMX6Q IMX6S"
+EXCLUDE_LIST:="IMX21ADS IMX27ADS IMX31ADS IMX31_3STACK IMX35_3STACK IMX25_3STACK IMX233 IMX50 IMX6Q IMX6SL IMX6SX"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/scc2_test.out
diff --git a/test/mxc_v4l2_test/Makefile b/test/mxc_v4l2_test/Makefile
index fee9c1e..33944cc 100644
--- a/test/mxc_v4l2_test/Makefile
+++ b/test/mxc_v4l2_test/Makefile
@@ -1,7 +1,7 @@
 # list of platforms which did not want this test case
 EXCLUDE_LIST:=
-CSI_LIST:=IMX25_3STACK IMX6S
-NEED_PXP_LIB_LIST:=IMX6S
+CSI_LIST:=IMX25_3STACK IMX6SL IMX6SX
+NEED_PXP_LIB_LIST:=IMX6SL IMX6SX
 
 # Just use pxp library to get contiguous physical memory for USERPTR testing
 # However, be noticed that this is not the only way.
diff --git a/test/pxp_lib_test/Makefile b/test/pxp_lib_test/Makefile
index c6895a4..4c68626 100644
--- a/test/pxp_lib_test/Makefile
+++ b/test/pxp_lib_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which want this test case
-INCLUDE_LIST:=IMX50 IMX51 IMX5 IMX6Q IMX6S
+INCLUDE_LIST:=IMX50 IMX51 IMX5 IMX6Q IMX6SL IMX6SX
 
 SRCS = pxp_test.c utils.c
 CFLAGS += -lpxp -lpthread
diff --git a/test/pxp_v4l2_test/Makefile b/test/pxp_v4l2_test/Makefile
index 746a166..ab9ae67 100644
--- a/test/pxp_v4l2_test/Makefile
+++ b/test/pxp_v4l2_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which want this test case
-INCLUDE_LIST:=IMX233 IMX28 IMX50 IMX51 IMX5 IMX6S
+INCLUDE_LIST:=IMX233 IMX28 IMX50 IMX51 IMX5 IMX6SL IMX6SX
 
 CFLAGS += -lpthread
 
-- 
1.8.0

